onnxruntime/python/tools/automl/feed_inputs_test.py

```diff
@@ -0,0 +1,38 @@
+#-------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#--------------------------------------------------------------------------
+
+import argparse
+import onnxruntime as onnxrt
+import numpy as np
+import pandas as pd
+from feed_inputs import DataFrameTool
+import os
+import sys
+
+def main():
+    parser = argparse.ArgumentParser(description='Test Feed Inputs utility')
+    parser.add_argument('model_path', help='model path')
+    parser.add_argument('-debug', action='store_true',
+                        help='pause execution to allow attaching a debugger.')
+    parser.add_argument('-profile', action='store_true',
+                        help='enable chrome timeline trace profiling.')
+    args = parser.parse_args()
+
+    # Load model
+    sess_options = onnxrt.SessionOptions()
+    sess_options.enable_profiling = args.profile
+
+    sess = onnxrt.InferenceSession(args.model_path, sess_options)
+
+    # Create a DataFrame that holds 3 inputs, string, bool, float in their respective columns
+    df = pd.DataFrame([['string_input', True, np.float32(0.25)]], index=[0], columns=['F2', 'Label', 'F1'])
```

Comment:
>np.float32(0.25)] [](start = 46, length = 17)

Panda converts this to in64 and displays erratic behavior with respect to types. @David Brownell is it still a good ide to do type checks?

Reply:
This seems to be a known issue: https://github.com/pandas-dev/pandas/issues/12255

I didn't read the entire thread, but is this something that we can work around until the issue is addressed properly in pandas?